src/renderers/server-render.jsx

```diff
@@ -184,10 +178,15 @@ const makeRenderer = (
 	const store = createStore(reducer, initialState);
 
 	// load initial config
-	store.dispatch(configureApiUrl(apiUrl));
-	store.dispatch(configureBaseUrl(host));
-	store.dispatch(configureLocaleCode(localeCode));
-
+	store.dispatch(
+		configure({
+			apiUrl,
+			baseUrl: host,
+			localeCode,
+			supportedLocaleCodes,
+			initialNow: new Date().getTime(),
```

Comment:
Is there a test case that could be updated to at least check for this value, and possibly test whatever date format this is returning?